client/foundries.go

```diff
@@ -174,6 +174,19 @@ type TufCustom struct {
 	OverridesSha   string                `json:"meta-subscriber-overrides-sha,omitempty"`
 }
 
+type ComposeAppContent struct {
+	Files       []string               `json:"files"`
+	ComposeSpec map[string]interface{} `json:"compose_spec"`
+}
+
+type ComposeAppInspect struct {
```

Comment:
Let's call this a ComposeAppBundle. There's some history where that's what we've called it.

Reply:
no problem, will change this